app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players: [Player]) -> int:
```

Comment:
Fix in all other similar cases

```suggestion
def calculate_team_total_rating(players: list[Player]) -> int:
```